scripts/docker/install_packages

```diff
@@ -17,13 +16,29 @@
 # specific language governing permissions and limitations
 # under the License.
 
-FILES_FOR_REBUILD_CHECK=(
- "setup.py"
- "setup.cfg"
- "Dockerfile"
- "Dockerfile-checklicence"
- ".dockerignore"
- "airflow/version.py"
- "airflow/www/package.json"
- "airflow/www/package-lock.json" )
-export FILES_FOR_REBUILD_CHECK
+# Bash sanity settings (error on exit, complain for undefined vars, error when pipe fails)
+set -euo
+
+. /etc/os-release
+
+if [ "${ID}" = "debian" ]; then
+    echo
+    echo "Installing packages for Debian:" "$@"
+    echo
+    apt-get update \
+    && apt-get install -y --no-install-recommends \
+           "$@" \
+    && apt-get autoremove -yqq --purge \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+elif [ "${ID}" = "alpine" ]; then
+    echo
+    echo "Installing packages for Alpine:" "$@"
+    echo
+    apk add --no-cache --virtual .build-deps "$@"
+else
+    echo
+    echo "Exiting. The os ${ID} is unknown."
+    echo
+    exit 1
+fi
```

Comment:
do we need to make one of these for darwin or is the assumption that mac users will build this part themselves for local dev?

Reply:
My bad.  I forgot to remove that file completely. It's not used now (no alpine image built at all). It was supposed to run as part of Docker build so it would never run on Darwin.